src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -203,7 +203,7 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
       // Extract the labels as the last dimension of the training data.
       Log::Info << "Using the last dimension of training set as labels."
           << endl;
-      labelsIn = conv_to<Row<size_t>>::from(
+      labelsIn = ConvTo<arma::Row<size_t>>::from(
```

Comment:
Technically if you're converting to something that's always an Armadillo object, you can just use `arma::conv_to`, but I can also see the advantage of replacing it like you did here.